src/cirrus/plugins/management/deployment.py

```diff
@@ -14,6 +18,8 @@
 MAX_SQS_MESSAGE_LENGTH = 2**18  # max length of SQS message
 CONFIG_VERSION = 0
 
+WORKFLOW_POLL_INTERVAL = 5  # seconds between state checks
```

Comment:
This feels like it might be better as an option on command? Some workflows might take 10 seconds, and others maybe hours, so I am not sure this is a one-size-fit-all type of parameter.